content/docs/examples/tutorial/setup-kubernetes-cluster/index.md

```diff
@@ -0,0 +1,208 @@
+    You can try using the [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart) or the
+    [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-getting-started).
+
+1.  Create an environment variable to store the name of a namespace to perform the commands of the tutorial on.
+    You can use any name, for example `tutorial`. `coolstuff` will do as well.
+
+    {{< text bash >}}
+    $ export NAMESPACE=tutorial
+    {{< /text >}}
+
+1.  Create the namespace:
+
+    {{< text bash >}}
+    $ kubectl create namespace $NAMESPACE
+    {{< /text >}}
+
+    {{< tip >}}
+    If you run the tutorial for multiple participants (you are an instructor in a class or you want to complete this
+    tutorial with your friends using your cluster), you may want to allocate a separate namespace per each
+    participant. The tutorial supports work in multiple namespaces simultaneously by multiple participants.
+    {{< /tip >}}
+
+1.  Install Istio with strict mutual TLS enabled. The simplest way is to follow
+    [these instructions](/docs/setup/kubernetes/install/kubernetes/#installation-steps),
+    select the `strict mutual TLS` tab.
+
+1.  [Enable Envoy's access logging](/docs/tasks/telemetry/logs/access-log/#enable-envoy-s-access-logging).
+
+1.  Create a Kubernetes Ingress resource for common Istio services:
+
+    - [Grafana](https://grafana.com/docs/guides/getting_started/)
+    - [Jaeger](https://www.jaegertracing.io/docs/1.13/getting-started/)
+    - [Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/)
+    - [Kiali](https://www.kiali.io/documentation/getting-started/)
+
+    If you do not know what are the services above - great, this tutorial will teach you about each of them.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: extensions/v1beta1
+    kind: Ingress
+    metadata:
+      name: istio-system
+      namespace: istio-system
+    spec:
+      rules:
+      - host: my-istio-dashboard.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: grafana
+              servicePort: 3000
+      - host: my-istio-tracing.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: tracing
+              servicePort: 80
+      - host: my-istio-logs-database.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: prometheus
+              servicePort: 9090
+      - host: my-kiali.io
+        http:
+          paths:
+          - path: /
+            backend:
+              serviceName: kiali
+              servicePort: 20001
+    EOF
+    {{< /text >}}
+
+1.  You may want to limit the permissions of each participant so they will be able to create resources only in their
+    namespace. Even if you run the tutorial alone on your own cluster, it could be a good practice to limit yourself so
```

Comment:
@rcaballeromx I rephrased the sentence a bit in https://github.com/istio/istio.io/pull/4704/commits/1bf6a42acc31a99f744199b72f5081fac656c5c0. Please note that sometimes I feel I need to be more modest and instead of telling the reader "follow this best practice", I prefer to say "It is a good practice".